ansible/roles/cloudwatch-agent/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
+=========
+
+A brief description of the role goes here.
```

Comment:
Let's fix this README up in a follow-up pull request, I will create a GitHub issue to cover this work @MadalinBorodi